src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-uk.gov.companieshouse.scanupondemand.orders.api.health=/healthcheck
+uk.gov.companieshouse.scanupondemand.orders.api.health=/orderable/scan-upon-demand/healthcheck
 uk.gov.companieshouse.scanupondemand.orders.api.home=/orderable/scan-upon-demand
 
 spring.data.mongodb.uri: ${MONGODB_URL}
```

Comment:
We will want to configure the `NoIsSnakeCaseFieldNamingStrategy` here as we do in the other items APIs so that we have fields in the database with names like `company_name` rather than `companyName`.

Reply:
done!

Reply:
I don't see it configured in this file yet?

Reply:
I see it now! 👍